### PR TITLE
Created Solution for "LC - Make Elements Equal"

### PR DESCRIPTION
**Pull request by dutinmeow**

### CODE REVIEW

File: solutions/lc-MakeElementEqual. [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+Because the problem does not specify which value we have to make all elements, note that only the relative values of the elements matter. Thus, increasing $n - 1$ b elements by one is the same as decreasing a single element by one. Solving this simplified problem is a matter of finding the minimum value in the array and transforming all other elements to match it, requiring a total of $sum - n \cdot min$ where $sum$ is the sum of the array and $min$ is the minimum value.
+
+**Time Complexity:** $\mathcal{O}(N)$
+
+<LanguageSection>
+<CPPSection>
+```c++
+class Solution {
+public:
```
Comment (SansPapyrus683) [inline solutions/lc-MakeElementEqual.:20]: is it convention to not indent the `public`? i always have but idk

File: solutions/lc-MakeElementEqual. [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+## Solution
+
+Because the problem does not specify which value we have to make all elements, note that only the relative values of the elements matter. Thus, increasing $n - 1$ b elements by one is the same as decreasing a single element by one. Solving this simplified problem is a matter of finding the minimum value in the array and transforming all other elements to match it, requiring a total of $sum - n \cdot min$ where $sum$ is the sum of the array and $min$ is the minimum value.
+
+**Time Complexity:** $\mathcal{O}(N)$
+
+<LanguageSection>
+<CPPSection>
+```c++
+class Solution {
```
Comment (SansPapyrus683) [inline solutions/lc-MakeElementEqual.:19]: add #include's?
idk how leetcode works

**Review by SansPapyrus683 — COMMENTED** [review]
ambivalent, going to abstain from approving

File: solutions/lc-MakeElementEqual. (hunk 1 above)
Comment (HarryWangATX) [reply solutions/lc-MakeElementEqual.:20]: its leetcode specific

File: solutions/lc-MakeElementEqual. (hunk 2 above)
Comment (HarryWangATX) [reply solutions/lc-MakeElementEqual.:19]: no need, leetcode specific as well

Comment (dutinmeow) [reply solutions/lc-MakeElementEqual.:19]: Leetcode is weird, apparently this is normal

File: solutions/lc-MakeElementEqual. (hunk 1 above)
Comment (SansPapyrus683) [reply solutions/lc-MakeElementEqual.:20]: wrong thread?

File: solutions/lc-MakeElementEqual. [hunk 3]
```diff
@@ -0,0 +1,32 @@
+---
```
Comment (maggie-j-liu) [inline solutions/lc-MakeElementEqual.:1]: did you forget the file extension 👀

Comment (dutinmeow) [reply solutions/lc-MakeElementEqual.]: oop, should be fixed now

**Comment by dutinmeow** [comment]
Problem was removed apparently ¯\_(ツ)_/¯

**Pull request closed without merging**